teamB/YaeRin/app.js

```diff
@@ -0,0 +1,53 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
+});
+
+app.post("/users/sign-up", async (req, res) => {
+  const { name, age } = req.body;
+  await myDataSource.query(
+    `INSERT INTO users(
+      name, 
+      age
+      ) VALUES (?, ?);
+      `,
+    [name, age]
+  );
+  res.status(201).json({ message: "New User Created!" });
+});
```

Comment:
변수 선언부와 함수 호출부 사이에 한 줄씩 띄워서 가독성을 높여주세요.

```suggestion
  const { name, age } = req.body;
  
  await myDataSource.query(
    `INSERT INTO users(
      name, 
      age
      ) VALUES (?, ?);
      `,
    [name, age]
  );
  
  res.status(201).json({ message: "New User Created!" });

});
```